common/app/views/support/Commercial.scala

```diff
@@ -160,7 +161,22 @@ object Commercial {
       classes mkString " "
     }
 
-    val slotCssClasses = Seq("top-banner-ad", "top-banner-ad-desktop")
+    def slotCssClasses(implicit request: RequestHeader): Seq[String] = {
```

Comment:
I believe `slotCssClasses` can stay a `val` and doesn't need to be turned into a function.

https://github.com/guardian/frontend/blob/5b970cd7308175cfc1bcae2d4fb8c06ee13c5fa0/common/app/views/fragments/commercial/topBanner.scala.html#L7-L11
`topAboveNavSlot.slotCssClasses` gets called here and still works because Scala allows a function to be called without `()` if it takes no arguments, but I'd leave it as a `val` just to be consistent with `adSizes` which is also a `val`.

Reply:
I think `slotCssClasses` needs to be a function in order to accept the `request` parameter. Otherwise the calls to `isParticipating` complain that they can't find an HTTP Request Header.

Reply:
You're right - I had wrongly assumed that import on [line 1](https://github.com/guardian/frontend/blob/5b970cd7308175cfc1bcae2d4fb8c06ee13c5fa0/common/app/views/fragments/commercial/topBanner.scala.html#L1) would take care of that!